master/getting-started/kubernetes/tutorials/advanced-policy.md

```diff
@@ -46,6 +46,11 @@ kubectl expose --namespace=advanced-policy-demo deployment nginx --port=80
 
 #### Check using calicoctl
 
+  > **Note:**
+  >
+  > This requires the [calicoctl tool to be configured]({{site.baseurl}}/{{page.version}}/reference/calicoctl/setup/etcdv2).
+  > For example: `export ETCD_ENDPOINTS=http://10.96.232.136:6666`
```

Comment:
I think these tutorials are decoupled from our installation, and most people use the selfhosted manifest (which uses this IP). If we make this a walkthrough tutorial which includes setting up k8s with selfhosted manifest then that's fine, but I don't want to assume people are using our vagrant installation for all these tutorials (and our vagrant tutorials are kinda old, using skydns etc.)